modules/installation-configuration-parameters.adoc

```diff
@@ -724,6 +728,10 @@ endif::openshift-origin[]
 |A pre-existing AWS IAM role applied to the control plane machine pool instance profiles. You can use these fields to match naming schemes and include predefined permissions boundaries for your IAM roles. If undefined, the installation program creates a new IAM role.
 |The name of a valid AWS IAM role.
 
+|`controlPlane.platform.aws.rootVolume.kmsKeyARN`
+|The Amazon Resource Name (key ARN)) of a KMS key. This is required to encrypt cluster master nodes OS volumes with specific KMS key.
+|Valid link:https://docs.aws.amazon.com/kms/latest/developerguide/find-cmk-id-arn.html[key ID and the key ARN].
```

Comment:
Suggest `cluster master nodes OS volumes` -> `OS volumes of cluster master nodes`.

Reply:
Addressed